### DISCUSSION TITLE
ViewSet HTTP response description

### DISCUSSION THREAD

**Discussion by LiorA1** [encode/django-rest-framework#7865] [Question & Answer]
Hi,

Why the Docs doesnt elaborate about the different possible responses?

Example:
User that upload blog successfully - return 200
User without permissions   - return 400/1

And so on ..

**Comment by lovelydinosaur** [comment]
Helping explain HTTP is such a huge subject, that it's not really clear where we'd start, or how we'd structure the docs.

I can imagine the docs doing better in this area, but it would be a huge amount of work. The closest we have at the moment is having plenty of[ links out to third party talks and tutorials](https://www.django-rest-framework.org/community/tutorials-and-resources/), which are free to cover the subject from different angles.

**Reply by LiorA1** [reply]
I didnt meant all of the HTTP subject. Just a small bit (maybe collapsible) that states primary Response codes for a ViewSet.

Not something wide

For Example: 
The ViewSet return Http_200 if everything have done correctly.
Http_401, if the server encounter auth problems.
